apiserver/facades/client/action/run.go

```diff
@@ -81,9 +81,6 @@ func (a *ActionAPI) Run(run params.RunParams) (results params.ActionResults, err
 	if err := a.checkCanAdmin(); err != nil {
 		return results, err
 	}
-	if err := a.check.ChangeAllowed(); err != nil {
```

Comment:
`juju run` is different. We should still block this. We only want to open up actions.

Reply:
Fixed